terraform/cognito.tf

```diff
@@ -2,12 +2,6 @@ resource "aws_cognito_user_pool" "pnp_user_pool" {
   name = "pnp-app-user-pool"
 
   auto_verified_attributes = ["email"]
-  schema {
-    attribute_data_type = "String"
-    name                = "email"
-    required            = true
-    mutable             = true
-  }
```

Comment:
The login method for the Cognito user pool still needs to be set up "email" instead of "name"